maps/away/away_site/unathi_pirate/tarwa/unathi_pirate_tarwa_ghostroles.dm

```diff
@@ -56,13 +56,29 @@
 		/obj/item/storage/box/survival = 1
 	)
 
+/datum/outfit/admin/tarwa/post_equip(mob/living/carbon/human/H, visualsOnly)
+	if(!istype(H))
+		return
+	for(var/name in H.organs_by_name)
+		var/obj/item/organ/external/O = H.organs_by_name[name]
+		if(!O || name == BP_HEAD || name == BP_CHEST || name == BP_GROIN)
+			continue
```

Comment:
You have an issue here with the loop variable `name` overlapping with the ghostspawner `name` variable.